src/components/TransactionHistoryRows/components/AssetsTransfers.tsx

```diff
@@ -0,0 +1,62 @@
+import { Box, Circle, Stack, Text as CText } from '@chakra-ui/react'
+import { useMemo } from 'react'
+import { Amount } from 'components/Amount/Amount'
+import { useButtonStyles } from 'hooks/useButtonStyles/useButtonStyles'
+import type { Transfer } from 'hooks/useTxDetails/useTxDetails'
+import { bn, bnOrZero } from 'lib/bignumber/bignumber'
+import { fromBaseUnit } from 'lib/math'
+
+import { FALLBACK_PRECISION } from '../constants'
+
+type AssetsTransfersProps = {
+  compactMode?: boolean
+  transfers: Transfer[]
+  index: number
+}
+
+export const AssetsTransfers: React.FC<AssetsTransfersProps> = ({
+  index,
+  compactMode,
+  transfers,
+}) => {
+  const { bg: circleBgColor, color: circleColor } = useButtonStyles({
+    colorScheme: 'blue',
+    variant: 'ghost-filled',
+  })
+
+  const aggregatedFiatValue = useMemo(
+    () =>
```

Comment:
linting creates extra newlines when not wrapped in `{return}`, I only use the shorthand return when it keeps it on a single line. just pref to reduce lines and increase readability imo.

another suggestion:
```
  const aggregatedFiatValue = useMemo(() => {
    const fiatValue = transfers.reduce((acc, transfer) => {
      const precision = transfer.asset?.precision ?? FALLBACK_PRECISION
      const cryptoValue = fromBaseUnit(transfer.value, precision)
      return acc.plus(bnOrZero(cryptoValue).times(transfer.marketData.price))
    }, bn(0))
    return fiatValue.toString()
  }, [transfers])
```
removes 3 lines of weird breaks

Reply:
Might just be years of aggressive Prettier refactor turning into linter Stockholm syndrome, but both are as readable for me 👀 . Refactored into explicit return syntax to make it more readable for everyone! f582041d19d9cc0b9d71a141252c6ca53127dfc6

Reply:
Still with a line break before the return statement, might as well have a bit of padding if using curly braces